src/ripple/app/ledger/impl/InboundLedgers.cpp

```diff
@@ -99,7 +100,8 @@ class InboundLedgersImp
         {
             ScopedLockType sl (mLock);
 
-            hash_map<uint256, InboundLedger::pointer>::iterator it = mLedgers.find (hash);
+            auto it = mLedgers.
+            find (hash);
```

Comment:
I applaud the style experiment.  If experiments never failed, they wouldn't be experiments.  This one failed.  I agree, this fails my (admittedly subjective) "readability" test.